docker/jenkins/periodic_db_update.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+# Needs DEIS_CONTROLLER URL, DEIS_USERNAME, DEIS_PASSWORD,
+# DEIS_PROFILE, and DEIS_APPLICATION environment variables.
+#
+# To set them go to Job -> Configure -> Build Environment -> Inject
+# passwords and Inject env variables
+#
+
+deis whoami || deis login $DEIS_CONTROLLER --username=$DEIS_USERNAME --password=$DEIS_PASSWORD
```

Comment:
I'd avoid this micro-optimization to limit the possibility of errors due to false or outdated logins and instead I'd always `deis login`